### ISSUE TITLE
Format of Password File

### ISSUE THREAD

**Issue by boxofscraps** [paritytech/parity#8230]
> I'm running:
>
> - **Which Parity version?**: 1.9.5-ff821daf1da42865f229aee35f2e74e7b2dd8db2
> - **Which operating system?**: Windows 10
> - **How installed?**: via installer
> - **Are you fully synchronized?**: yes
> - **Which network are you connected to?**: kovan
> - **Did you try to restart the node?**: yes

I am getting "Unable to read password file. Ensure it exists and permissions are correct." error when trying to unlock my account: 

I have verified that the account I am trying to unlock is listed under: parity account list

I have tried what was explained here: https://github.com/paritytech/parity/issues/6615
In my password.file, I current have: password  (on the first line, no extra spaces in plaintext)
I have also tried password with quotes. 

Thank you.

**Comment by 5chdn** [comment]
Please share how you are passing the file to parity.

**Comment by boxofscraps** [comment]
Thanks for the quick response. 
I used the toml configuration and my config.toml has the following. 

chain = "kovan"
[account]
unlock = ["0xADDRESS"]
password = ["%UserProfile%/password.txt"]

I then used the command line from %UserProfile%. echo password > password.file

**Comment by Tbaut** [comment]
I answered you in https://github.com/paritytech/parity/issues/8218
I guess the problem is the path to your password file. Make sure it doesn't contain any macro such as %UserProfile% or $HOME. I just tested it successfully with relative and direct path. I'll update the wiki.

**Comment by boxofscraps** [comment]
Yes, I appreciate your help in https://github.com/paritytech/parity/issues/8218. Unfortunately, I still have an issue with unlocking my account. I have attempted at reading every github issue or stackoverflow question regarding unlock parity but still cannot figure out what to do. I feel very frustrated at myself as everyone else has been able to do it without so much assistance. 

Previously, I was getting the Invalid type: string error (expected a sequence for key account.password).  This was fixed by placing the password location in an array. 

Now I am getting "Unable to read password file. Ensure it exists and permissions are correct." Error which I think is caused by the password file being wrong.  File path with relative and direct path both have the same error.
